packages/mattermost-redux/src/selectors/entities/preferences.ts

```diff
@@ -112,7 +111,7 @@ export const getTeammateNameDisplaySetting: (state: GlobalState) => string = cre
 
 const getThemePreference = createSelector(
     getMyPreferences,
-    getCurrentTeamId,
+    (state) => state.entities.teams.currentTeamId,
```

Comment:
Why is this changing?

Reply:
@hmhealey to avoid import loop caused by getCurrentTeamId import

Reply:
Ah, I see. We could also solve that by moving `getCurrentTeamId` into `selectors/common` like we did with `getCurrentUserId` and some others before, but I'm good with it either way.